infrastructure/main.tf

```diff
@@ -31,3 +31,21 @@ resource "azurerm_key_vault_secret" "redis_access_key" {
   value        = module.nfdiv-frontend-session-storage.access_key
   key_vault_id = data.azurerm_key_vault.key_vault.id
 }
+
+module "nfdiv-frontend-redis6" {
+  source   = "git@github.com:hmcts/cnp-module-redis?ref=master"
+  product  = "${var.product}-${var.component}-redis"
```

Comment:
this should be var.product then you should have a second attribute being passed in:

name = "${var.product}-${var.env}"